test/fixtures/framework/features/support/env.rb

```diff
@@ -0,0 +1,4 @@
+MazeRunner.hooks.before do |scenario|
```

Comment:
I think this would be more useful if it checked the different kind of hooks, when they are called and the order that two of the same type of hook are called

Reply:
Yes, I agree - I'll do this as a follow-up PR straight afterwards, if that's ok, just so that we can get a release out.